### PR TITLE
initialize_user.sh to take in flag for dev

### PR DESCRIPTION
**Pull request by fanpu**
## Description

Let ./initialize_user.sh to take in a flag for dev (-d), so that people running dev installations can also get a user created (by default it uses prod env vars)

## How Has This Been Tested?

./bin/initialize_user.sh -d works for creating a dev user
./bin/initialize_user.sh will also work once https://github.com/autolab/Autolab/pull/1363 is merged

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My change requires a change to the documentation, which is located at [Autolab Docs](https://github.com/autolab/docs)
- [x] I have updated the documentation accordingly, included in this PR

## Other issues / help required

If unsure, feel free to submit first and we'll help you along.

### CODE REVIEW

File: docs/installation/osx.md [hunk 1]
```diff
@@ -86,11 +86,15 @@ Follow the step-by-step instructions below:
 
     Do not forget to use `bundle exec` in front of every rake/rails command.
 
-12. Create initial root user
+12. Create initial root user. (Flag is required for differentiating the environment because this affects the database (prod or dev) in which the user is created.)
```
Comment (oliverli) [inline docs/installation/osx.md:89]: Nit: just rephrase this a little bit (e.g. "pass the `-d` flag for developmental deployments). Not a blocker though

**Review by oliverli — DISMISSED** [review]
Documentation could be phrased a little clearer imo in classic bash `--help` style or similar, but not a blocker

File: docs/installation/osx.md (hunk 1 above)
Comment (fanpu) [reply docs/installation/osx.md]: Done, could you approve this again? Thanks!

**Review by oliverli — APPROVED** [review]
Lgtm!

**Pull request merged**